automation_tools/satellite6/upgrade/satellite.py

```diff
@@ -0,0 +1,153 @@
+import os
+import sys
+import time
+
+from tools import reboot
```

Comment:
Where tools package comes from? If this is a relative import, drop that in favor of absolute import.